Tests/AK/TestCharacterTypes.cpp

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2018-2020, Andreas Kling <kling@serenityos.org>
```

Comment:
Should be you? Also the copy right year needs to be fixed.

Reply:
Oops :^)